lib/openxml/drawingml/elements.rb

```diff
@@ -0,0 +1,13 @@
+module OpenXml
+  module DrawingML
+    module Elements
+    end
+  end
+end
+
+require "openxml/docx/elements/element"
+require "openxml/docx/elements/container"
+require "openxml/docx/elements/text"
```

Comment:
Yes; they're the base classes for generic XML elements, container elements, and text nodes. These are things that could potentially be pulled down to `OpenXml::Package` if we spin off `OpenXml::DrawingML` into its own gem.